x-pack/metricbeat/mbreceiver/receiver_test.go

```diff
@@ -110,3 +169,13 @@ func TestMultipleReceivers(t *testing.T) {
 		},
 	})
 }
+
+func genSocketPath() string {
+	randData := make([]byte, 16)
+	for i := range len(randData) {
+		randData[i] = uint8(rand.UintN(255)) //nolint:gosec // 0-255 fits in a unint8
```

Comment:
```suggestion
		randData[i] = uint8(rand.UintN(255)) //nolint:gosec // 0-255 fits in a uint8
```

Reply:
🤦 thanks for catching that, fixed in f464cdf